client/src/components/UserContacted.jsx

```diff
@@ -9,19 +9,19 @@ import {
 
 function UserContacted({ user, userContactedOnClick }) {
   return (
-    <ListItem button onClick={() => userContactedOnClick(user.id)}>
+    <ListItem button onClick={() => userContactedOnClick(user.userId)}>
       <ListItemIcon>
-        <Avatar alt={user.senderName} src="" />
+        <Avatar alt={user.senderName} src={user.image} />
```

Comment:
Should we setup a fallback/placeholder image just in case the image in the database is missing